data/src/main/scala/org/apache/predictionio/data/store/LEventStore.scala

```diff
@@ -72,9 +84,61 @@ object LEventStore {
     latest: Boolean = true,
     timeout: Duration = defaultTimeout): Iterator[Event] = {
 
+    import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Since using `ExecutionContext.Implicits.global` is bad habit in Scala basically, I wanted to make sure to use it only in these methods.

And your interpretation is correct. Even if we call it twice, it returns a same instance.